src/styles/index.scss

```diff
@@ -1,3 +1,121 @@
+@import './stars';
+@import './variables';
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto , sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: $white-color;
+  width: 200px;
+  height: $card-height;
+  flex-shrink: 0;
+  justify-content: center;
+  justify-items: center;
```

Comment:
redundant - first 2 of them doesn't work at all w/o display: flex, the last one does nothing